clients/elixir/lib/Vexilla/scheduler.ex

```diff
@@ -52,76 +43,81 @@ defmodule Scheduler do
         start_time = Timex.from_unix(schedule["startTime"], :millisecond)
         end_time = Timex.from_unix(schedule["endTime"], :millisecond)
 
-        case {schedule["timeType"],
-              Timex.between?(now, start_of_start_date, end_of_end_date, inclusive: true)} do
-          {^time_type_none, true} ->
-            true
-
-          {^time_type_start_end, true} ->
-            start_day_with_start_time =
-              Timex.set(start_date,
-                hour: start_time.hour,
-                minute: start_time.minute,
-                second: start_time.second,
-                microsecond: start_time.microsecond
-              )
-
-            end_day_with_start_time =
-              Timex.set(end_date,
-                hour: end_time.hour,
-                minute: end_time.minute,
-                second: end_time.second,
-                microsecond: end_time.microsecond
-              )
-
-            Timex.between?(now, start_day_with_start_time, end_day_with_start_time,
-              inclusive: true
-            )
-
-          {^time_type_daily, true} ->
-            zero_day = Timex.from_unix(0)
-            now_timestamp = Timex.to_unix(now)
-
-            today_zero_timestamp =
-              Timex.set(now, hour: 0, minute: 0, second: 0, microsecond: 0)
-              |> Timex.to_unix()
-
-            zeroed_start_timestamp =
-              Timex.set(zero_day,
-                hour: start_time.hour,
-                minute: start_time.minute,
-                second: start_time.second,
-                microsecond: start_time.microsecond
+        if Timex.between?(datetime, start_of_start_date, end_of_end_date, inclusive: true) do
```

Comment:
I think I would prefer not to use an if statement here, if only because it makes the diff noisier than it probably should be.

A later PR could refactor this logic.